src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/NuGetProjectService.cs

```diff
@@ -95,7 +96,19 @@ NuGetInstalledPackage ToNuGetInstalledPackage(PackageReference packageReference,
             IReadOnlyCollection<NuGetInstalledPackage> installedPackages;
 
             var cacheContext = new DependencyGraphCacheContext();
-            var (packageSpecs, messages) = await project.GetPackageSpecsAndAdditionalMessagesAsync(cacheContext);
+            IReadOnlyList<ProjectModel.PackageSpec> packageSpecs;
+            IReadOnlyList<ProjectModel.IAssetsLogMessage> messages;
+            try
+            {
+                (packageSpecs, messages) = await project.GetPackageSpecsAndAdditionalMessagesAsync(cacheContext);
+            }
+            catch (ProjectNotNominatedException)
+            {
+                status = InstalledPackageResultStatus.ProjectNotReady;
+                installedPackages = null;
+                return (status, installedPackages);
```

Comment:
```suggestion
                return (status, null);
```

Reply:
I feel like this violates our coding guidelines where we expect types/variables to be obvious. When calling methods with a null parameter, we usually name the parameter, so people reading know what the null represents. Here, by assigning a null value to the variable and then using the variable in the tuple, it's extremely clear to anyone reading the code what the two values represent.